docs/container_setup/github_codespace/creating_codespace.md

```diff
@@ -19,3 +19,27 @@ GitHub Codespaces' button and then click the green 'Create Codespace' button.
 
 3. You will be taken to a VSCode editor within your browser.
 ![codespace](../../assets/rdev3.png)
+
+4. Rename your codespace for easier identification:
+
+By default, your codespace will have a randomly generated name. Renaming it to
+something descriptive — such as referencing the issue or feature you’re working
+on makes it much easier to find and manage your codespaces, especially if
+you use several.
+
+To rename your codespace:**
+
+- Open your [GitHub Codespaces dashboard](https://github.com/codespaces) in
+  your browser.
+- Find the codespace you just created (it will show the repository name and
+  the current random display name).
+- Click the ellipsis (**...**) next to your codespace to open the options menu.
+- Select **Rename** from the dropdown.
+- Enter a descriptive new display name (e.g., `r-dev-env-issue-229` or
+  `r-dev-env-patch-naming`) and confirm.
```

Comment:
```suggestion
- Enter a descriptive new display name, e.g., `Bug 7084 - text() recycling problems` or `Ignore deprecated/defunct fns in checkS3methods`, and confirm.
```

Remember the container is intended for working on bugs/features in R, not issues with the R dev container. It is not necessary to include `r-dev-env` in the name, as this is already shown (as you point out in the second bullet point here).